R/cancensus.R

```diff
@@ -325,7 +325,21 @@ cancensus.list_vectors <- function(dataset) {
   result %>%
     dplyr::mutate(type = factor(type),
                   units = factor(units)) %>%
-    dplyr::select(vector, type, label, units, parent_vector = parent)
+    dplyr::select(vector, type, label, units, parent_vector = parent) %>%
+    dplyr::mutate(
+      units = recode(.$units, 
+                         `1` = "number",
+                         `2` = "percentage ratio (0.0-1.0)",
+                         `3` = "currency",
+                         `4` = "ratio",
+                         `5` = "percentage (0-100)")) %>%
+    dplyr::mutate(
+      add = recode(.$add,
+                         `1` = "Additive",
+                         `2` = "Averages",
```

Comment:
This won't parse properly, if the value starts with "2" it will be followed by ".xxxx" where "xxxx" is another vector that should be used as weights to form a weighted sum when adding this. I could also just spit this data up and send it down as another column, say "base".